src/test_utils/index.tsx

```diff
@@ -0,0 +1,94 @@
+import {
```

Comment:
I hate to have such a duplicate of a Spruce file 😩 maybe down the line we can think about how best to share this code. But looks good for now!

Reply:
😔😭👍